apps/mudita-center-e2e/src/specs/overview/kompakt-switching-devices.ts

```diff
@@ -0,0 +1,141 @@
+    })
+
+    await browser.pause(6000)
+    const menuItem = await $(`//a[@href="#/generic/mc-overview"]`)
+
+    await menuItem.waitForDisplayed({ timeout: 10000 })
+    await expect(menuItem).toBeDisplayed()
+  })
+
+  it("Verify Overview Page", async () => {
+    const kompaktImage = await OverviewKompaktPage.kompaktImage
+    await expect(kompaktImage).toBeDisplayed
+    console.log(kompaktImage)
+    await expect(kompaktImage).toHaveAttribute("src", kompaktImageRegex)
+
+    const kompaktOsVersion = await OverviewKompaktPage.kompaktOsVersion
+    await expect(kompaktOsVersion).toBeDisplayed
+
+    const kompaktSimCard1 = await OverviewKompaktPage.kompaktSimCard1
+    await expect(kompaktSimCard1).toBeDisplayed
+    const sim1: string = "SIM 1"
+    const simInfo: string = sim1
+  })
+
+  it("Connect 2nd device", async () => {
+    E2EMockClient.mockResponse({
+      path: "path-2",
+      body: overviewDataWithOneSimCard2nd,
+      endpoint: "FEATURE_DATA",
+      method: "GET",
+      status: 200,
+    })
+    E2EMockClient.addDevice({
+      path: "path-2",
+      serialNumber: secondSerialNumber,
+    })
+
+    await browser.pause(6000)
+    const menuItem = await $(`//a[@href="#/generic/mc-overview"]`)
+
+    await menuItem.waitForDisplayed({ timeout: 10000 })
+    await expect(menuItem).toBeDisplayed()
+  })
+
+  it("Check Drawer Modal and Switch to 2nd device", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
+
+    const deviceImageOnDrawer = await drawerPage.deviceImageOnDrawer
+    await expect(deviceImageOnDrawer).toBeDisplayed()
+
+    const firstDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      firstSerialNumber
+    )
+    await expect(firstDeviceOnDrawer).toBeDisplayed()
+    const secondDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      secondSerialNumber
+    )
+    await expect(secondDeviceOnDrawer).toBeDisplayed()
+    await secondDeviceOnDrawer.waitForClickable()
+    await secondDeviceOnDrawer.click()
+  })
+
+  it("Verify Overview Page for 2nd device", async () => {
+    const kompaktImage = await OverviewKompaktPage.kompaktImage
+    await expect(kompaktImage).toBeDisplayed
+    console.log(kompaktImage)
+    await expect(kompaktImage).toHaveAttribute("src", kompaktImageRegex)
+
+    const kompaktOsVersion = await OverviewKompaktPage.kompaktOsVersion
+    await expect(kompaktOsVersion).toBeDisplayed
+
+    const kompaktSimCard1 = await OverviewKompaktPage.kompaktSimCard1
+    await expect(kompaktSimCard1).toBeDisplayed
+    const sim1: string = "SIM 1"
+    const simInfo: string = sim1
+  })
+
+  it("Verify Select Connected Devices, click on it and select 1st Kompakt", async () => {
+    const selectConnectedDevices = await overviewPage.selectConnectedDevices
+    await selectConnectedDevices.waitForClickable()
+    await expect(selectConnectedDevices).toHaveTextContaining("2")
```

Comment:
```suggestion
    await expect(selectConnectedDevices).toHaveText("2")
```
`toHaveTextContaining` will give false positive result for any string with "2" e.g. "22", "123", "te2t"